publishable/database/migrations/2016_01_01_000000_add_user_avatar_and_role_id.php

```diff
@@ -6,25 +6,23 @@ class AddUserAvatar extends Migration
 {
     /**
```

Comment:
However, maybe it is not so bad to have the migrations separated like before. Since the name explains exactly what is happening.

Thoughts?